samza-core/src/main/java/org/apache/samza/processor/SamzaContainerController.java

```diff
@@ -105,7 +105,9 @@ public void startContainer(ContainerModel containerModel, Config config, int max
         localityManager,
         new JmxServer(),
         Util.<String, MetricsReporter>javaMapAsScalaMap(metricsReporterMap),
-        taskFactory);
+        taskFactory,
+        // TODO: need to use the correct local ApplicationRunner here
+        null);
```

Comment:
Is this going to cause NPEs later right now? Maybe create a ticket to finish this wiring and add to the TODO?